core/src/test/java/org/infinispan/test/TestingUtil.java

```diff
@@ -695,8 +698,8 @@ public static void clearContent(EmbeddedCacheManager cacheContainer) {
          }
       }
 
-      if (cacheContainer.isDefaultRunning()) {
-         Cache defaultCache = cacheContainer.getCache();
+      Cache defaultCache = cacheContainer.getCache(EmbeddedCacheManager.DEFAULT_CACHE_NAME);
+      if (defaultCache != null) {
```

Comment:
We do return null if there was an exception during `cache.start()`, because we insert the `CacheWrapper` in the map of caches but we don't call `setCache`.

I did make a mistake here though, I should have kept the `isDefaultRunning()` check, otherwise it could start the default cache.